source/_docs/installation/synology.markdown

```diff
@@ -2,78 +2,177 @@
+1. Click on "*Next*"
+1. You will most likely get a unverified signature warning, click on "*Yes*"
+1. Click on "*Apply*"
+
+Now while it's installing, you may setup the Shared-Folder for Home Assistant:
+1. Open "[*Control Panel*](https://www.synology.com/en-global/knowledgebase/DSM/help/DSM/AdminCenter/ControlPanel_desc)"
+1. Go to "[*Shared-Folder*](https://www.synology.com/en-global/knowledgebase/DSM/help/DSM/AdminCenter/file_desc)" settings
+1. Click on "*Create*"
+1. In "*Name*" write "**homeassistant**"
+1. in "*Description*" write "**Home Assistant**"
+1. Click on "*Next*"
+1. Click on "*Next*" again
+1. Click on "*Apply*"
+
+Next setup the user:
+1. Open "[*Control Panel*](https://www.synology.com/en-global/knowledgebase/DSM/help/DSM/AdminCenter/ControlPanel_desc)"
+1. Go to "[*User*](https://www.synology.com/en-global/knowledgebase/DSM/help/DSM/AdminCenter/file_user_desc)" settings
+1. Click on "*Create*"
+1. In "*Name*" write "**homeassistant**"
+1. In "*Description*" write "**Home Assistant**"
+1. The "*Password*" is up to your choice
+1. Click on "*Next*"
+1. Make sure only "*users*" group is checked and click "*Next*"
+1. Set the permission "*Read/Write*" for "*homeassistant*" and all other Shared-Folders to "*No access*"
+1. Click on "*Next*"
+1. Click on "*Next*" again
+1. Click on "*Apply*"
+
+Next you need to enable SSH:
+1. Open "[*Control Panel*](https://www.synology.com/en-global/knowledgebase/DSM/help/DSM/AdminCenter/ControlPanel_desc)"
+1. Go to "[*Terminal & SNMP*](https://www.synology.com/en-global/knowledgebase/DSM/help/DSM/AdminCenter/system_terminal)" settings
+1. Click on the checkbox next to "*Enable SSH service*"
+1. Click on "*Apply*"
+
+If you turned on the firewall on your Synology NAS, please config it to allow connections for Home Assistant:
+1. Open "[*Control Panel*](https://www.synology.com/en-global/knowledgebase/DSM/help/DSM/AdminCenter/ControlPanel_desc)"
+1. Go to "[*Security*](https://www.synology.com/en-global/knowledgebase/DSM/help/DSM/AdminCenter/connection_security_desc)" settings
+1. Go to "[*Firewall*](https://www.synology.com/en-global/knowledgebase/DSM/help/DSM/AdminCenter/connection_security_firewall)" settings
+1. Go to "*Edit Rules*"
+1. Click on "*Create*"
+1. Select Custom: Destination port "TCP"
+1. Type "8123" in port (e.g., setting of "[*server_port*](/components/http/#server_port)" in configuration.yaml)
+1. Click on "*OK*"
+1. Click on "*OK*" again
+
+## {% linkable_title Installing Home Assistant %}
+
+After the Python 3 package has been installed, open terminal and open SSH to the synology.<br/>
+Replace "*user*" with your Synology NAS user and "x.x.x.x" with the its IP address:
+```bash
+ssh user@x.x.x.x
+```
+Use `pip` to install Home Assistant:
 ```bash
+# sudo /volume1/@appstore/python3/bin/python3 -m pip install homeassistant
+```
+Create a file named "**hass-daemon**" in your "*homeassistant*" Shared-Folder with the script below as its content.<br/>
```

Comment:
Create a file named `hass-daemon` in your `homeassistant` Shared-Folder with the script below as its content.<br/>